### DISCUSSION TITLE
Electric Tariff input values

### DISCUSSION THREAD

**Discussion by vingrirr** [NREL/REopt-API-Analysis#140] [8 - API: Application Programming Interface] [answered]
Hello,  I have a list of lat/long values and I am getting the commercial utility rate values from 
NREL's utility-rates-v3 (https://developer.nrel.gov/docs/electricity/utility-rates-v3/)

I am then calling the ReOpt API but I am unsure which of the ElectricTariff input value would correspond to the utlity rates API value?

Thanks,

**Comment by adfarth** [comment, marked as answer]
Hi! It looks like that API returns a single $/kWh rate. You could enter that $/kWh values into the `ElectricTariff: blended_annual_energy_rate ` input. If using that input you'll need to also supply a `blended_annual_demand_rate` (can set to 0.0). 

See the [REopt.jl documentation](https://nrel.github.io/REopt.jl/dev/reopt/inputs/#ElectricTariff) for other ways you can input rate details.

**Reply by vingrirr** [reply]
Thanks, much appreciated!
